test/ssltestlib.c

```diff
@@ -18,7 +18,14 @@
 # include <unistd.h>
 
 static ossl_inline void ossl_sleep(unsigned int millis) {
+# ifdef OPENSSL_SYS_VXWORKS
```

Comment:
while you're here, please fix the line above and put the open curly on a line by itself.